src/clsourcedependencies_parser.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
Note that this was the blocker to merging this PR earlier. See https://github.com/ninja-build/ninja/pull/1812#discussion_r456275065, #2153, and #2188.

Reply:
fixed by f6aa7537a7d003f17ebce2fdcb32bddf26fca305